lms/sentry.py

```diff
@@ -1,16 +1,16 @@
 """Sentry crash reporting integration."""
-from lms.services import ProxyAPIAccessTokenError
+from lms.services import OAuth2TokenError
 
 
-def filter_proxy_api_access_token_error(event):
-    """Filter out all ProxyAPIAccessTokenError's."""
-    return isinstance(event.exception, ProxyAPIAccessTokenError)
+def filter_oauth2_token_error(event):
```

Comment:
This hasn't change on the PR but the name and docstring made me pause for a second, no filtering going on here but rather at h_pyramid_sentry.filters